marketplacetests/tests/test_marketplace_search_for_paid_app.py

```diff
@@ -10,25 +10,13 @@ class TestSearchMarketplacePaidApp(MarketplaceGaiaTestCase):
 
     def test_search_paid_app(self):
```

Comment:
Note that I confirmed the change to this test with @krupa as it is not functionally equivalent to its predecessor. She said the new version of the test is fine.